nfl/storage/neo4j_store.py

```
@@ -4,8 +4,19 @@
 
 try:
     from neo4j import GraphDatabase
-except Exception:  # pragma: no cover - offline stub
-    from offline_stubs.neo4j import GraphDatabase
+except Exception:  # pragma: no cover - fallback stub
+    class _Session:
```

Pylint (3.3.7)

[refactor] Too few public methods (1/2)

(R0903)